### DISCUSSION TITLE
Sonoff NSPANEL update to release 13.0 fail because of size issues

### DISCUSSION THREAD

**Discussion by acdcnow** [arendst/Tasmota#18993] [Q&A]
Having right now Release 12.5 on sonoff NSPANEL

[tasmota32-nspanel.bin](https://github.com/arendst/Tasmota/releases/download/v12.5.0/tasmota32-nspanel.bin)
Filesize 1.49 MB Apr 17

Program Version | 12.5.0(nspanel)
2023-04-17T08:07:02
2_0_7/4.4.4.20230218
0T21:29:19

Upgrade to 13.0 via OTA link or file upload failed.
tasmota32-nspanel.bin | http://ota.tasmota.com/tasmota32/release/tasmota32-nspanel.bin | 1871k | 20230626 13:42

That does not fit any more because if the size difference. 
Why at all is there so much size needed, what has been added additionally or just a compile issue?

**Comment by Jason2866** [comment]
See https://github.com/arendst/Tasmota/discussions/18983
